lerna.json

```diff
@@ -7,7 +7,9 @@
       "changelogPreset": "conventionalcommits"
     }
   },
+  "includeMergedTags": true,
```

Comment:
What's the motivation for this commit? How does `includeMergedTags` accomplish it?